src/Microsoft.Azure.SignalR.Management/TaskExtensions.cs

```diff
@@ -31,11 +31,7 @@ private static async Task OrTimeoutCore(this Task task, CancellationToken cancel
                 return;
             }
 
-            var completed = await Task.WhenAny(task, Task.Delay(Timeout.InfiniteTimeSpan, cancellationToken));
-            if (completed != task)
-            {
-                throw new TimeoutException("Operation timed out");
-            }
+            await Task.WhenAny(task, Task.Delay(Timeout.InfiniteTimeSpan, cancellationToken));
```

Comment:
this looks like a breaking change, previously it throws, now it does not, will it impact any existing logic?

Reply:
reverted